src/Sulu/Bundle/ContactBundle/Resources/config/forms/contact_details.xml

```diff
@@ -46,24 +46,12 @@
                     </params>
                 </property>
 
-                <property name="formOfAddress" type="single_select" mandatory="true" colspan="3">
+                <property name="formOfAddress" type="single_select" colspan="3">
```

Comment:
I don't see why this field should be mandatory, because the database column isn't mandatory